src/Symfony/Component/Form/Tests/Extension/Core/DataTransformer/MoneyToLocalizedStringTransformerTest.php

```diff
@@ -72,4 +72,18 @@ public function testReverseTransformEmpty()
 
         $this->assertNull($transformer->reverseTransform(''));
     }
+
+    /**
+     * @requires extension bcmath
+     */
+    public function testFloatToIntConversionMismatchOnReversTransform()
+    {
+        $transformer = new MoneyToLocalizedStringTransformer(null, null, null, 100);
+
+        if (!function_exists('bcmul')) {
+            $this->markTestSkipped('Test will only work when bcmath-extension is enabled');
+        }
+        $this->assertEquals(3655, $transformer->reverseTransform('36.55'));
```

Comment:
I think I don't need this assertions because `$this->assertSame(3655, (int) $transformer->reverseTransform('36.55'));` should fullfill this case. I will remove it if you don't have objections.